pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -198,38 +211,137 @@ function setupTemplate($request) {
 			array('submissionId' => $submission->getId())
 		);
 
-		$submissionProps = Services::get('submission')->getFullProperties(
+		$titleAbstractForm = new PKP\components\forms\publication\PKPTitleAbstractForm($latestPublicationApiUrl, $locales, $latestPublication);
+		$citationsForm = new PKP\components\forms\publication\PKPCitationsForm($latestPublicationApiUrl, $latestPublication);
+
+		// Import constants
+		import('classes.submission.Submission');
+		import('classes.components.forms.publication.PublishForm');
+
+		$templateMgr->setConstants([
+			'STATUS_QUEUED',
+			'STATUS_PUBLISHED',
+			'STATUS_DECLINED',
+			'STATUS_SCHEDULED',
+			'FORM_TITLE_ABSTRACT',
+			'FORM_CITATIONS',
+		]);
+
+		// Get the submission props without the full publication details. We'll
+		// retrieve just the publication information that we need separately to
+		// reduce the amount of data passed to the browser
+		$propNames = Services::get('schema')->getSummaryProps(SCHEMA_SUBMISSION);
+		$propNames = array_filter($propNames, function($propName) { return $propName !== 'publications'; });
+		$submissionProps = Services::get('submission')->getProperties(
 			$submission,
+			$propNames,
 			[
 				'request' => $request,
-				'userGroups' => DAORegistry::getDAO('UserGroupDAO')->getByRoleId($submission->getData('contextId'), ROLE_ID_AUTHOR)->toArray(),
+				'userGroups' => $contextUserGroups,
 			]
 		);
 
-		$templateMgr->assign('workflowData', [
-			'components' => [
+		// Get an array of publication identifiers
+		$publicationList = [];
+		foreach ($submission->getData('publications') as $publication) {
+			$publicationList[] = Services::get('publication')->getProperties(
+				$publication,
+				['id', 'datePublished', 'status'],
+				[
+					'context' => $submissionContext,
+					'submission' => $submission,
+					'request' => $request,
+				]
+			);
+		}
 
+		// Get full details of the working publication and the currently published publication
+		$workingPublicationProps = Services::get('publication')->getFullProperties(
+			$submission->getLatestPublication(),
+			[
+				'context' => $submissionContext,
+				'submission' => $submission,
+				'request' => $request,
+				'userGroups' => $contextUserGroups,
+			]
+		);
+		if ($submission->getLatestPublication()->getId() === $submission->getCurrentPublication()->getId()) {
+			$currentPublicationProps = $workingPublicationProps;
+		} else {
+			$currentPublicationProps = Services::get('publication')->getFullProperties(
+				$submission->getCurrentPublication(),
+				[
+					'context' => $submissionContext,
+					'submission' => $submission,
+					'request' => $request,
+					'userGroups' => $contextUserGroups,
+				]
+			);
+		}
+
+		// Check if current author can edit metadata
+		$disableSave = true;
+		if (Services::get('submission')->canUserEditMetadata($submission->getId(), $user->getId()) ) {
+			$disableSave =  false;
```

Comment:
Let's call this `canEditPublication` here too.

Reply:
renamed